build/tools/Automation/Telemetry/LoggerImpl.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Oryx.Automation.Telemetry
+{
+    public class LoggerImpl : ILogger
```

Comment:
Similar to the `HttpClientImpl` comment -- what is the purpose of this class?

Reply:
I've converted Logger to an extensions class. I'm thinking of adding the telemetry code in the python PR